<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringPicker</class>
 <widget class="QWidget" name="StringPicker">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Picker</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_value">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_pickStr">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_value">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pick your value</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_value</sender>
   <signal>returnPressed()</signal>
   <receiver>button_pickStr</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
